contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -3004,6 +3004,15 @@ SELECT
 WHERE FALSE; -- This condition will ensure that the view is empty
 GRANT SELECT ON sys.numbered_procedures TO PUBLIC;
 
+ALTER FUNCTION sys.fn_mapped_system_error_list() RENAME TO fn_mapped_system_error_list_deprecated_in_2_2_0;
+
+CREATE OR REPLACE FUNCTION sys.fn_mapped_system_error_list ()
```

Comment:
we should add upgrade test where this is called from a view/function/procedure.

Reply:
done!